demo/src/main/java/com/novoda/demo/DemoPresenter.java

```diff
@@ -0,0 +1,120 @@
+package com.novoda.demo;
+
+import android.net.Uri;
+
+import com.novoda.noplayer.ContentType;
+import com.novoda.noplayer.Listeners;
+import com.novoda.noplayer.NoPlayer;
+import com.novoda.noplayer.PlayerState;
+import com.novoda.noplayer.PlayerView;
+import com.novoda.noplayer.model.VideoDuration;
+import com.novoda.noplayer.model.VideoPosition;
+
+class DemoPresenter {
+
+    private final ControllerView controllerView;
+    private final NoPlayer noPlayer;
+    private final Listeners listeners;
+    private final PlayerView playerView;
```

Comment:
Two views, I could use a displayer but to me displayers just introduce an unnecessary layer 🤔  but perhaps I don't fully understand MVP 😆